code/image_modifications.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>Nothing big, but maybe we could move the duplicate morphology import. </p><p>Also a lot of interesting methods I first have to dig into, but the result looks promising!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>